superset/__init__.py

```diff
@@ -222,9 +222,11 @@ def is_feature_enabled(feature):
 if conf.get("ENABLE_FLASK_COMPRESS"):
     Compress(app)
 
+
+talisman = Talisman()
```

Comment:
is there a cost to always initializing this, even if the config var isn't set?

Reply:
@etr2460 is virtually a no-op per [here](https://github.com/GoogleCloudPlatform/flask-talisman/blob/master/flask_talisman/talisman.py#L56), i.e., here's the contents of `__init__.py`:

```
class Talisman:
    def __init__(self, app=None, **kwargs):
            if app is not None:
                self.init_app(app, **kwargs)
```

which we're instantiating via `Talisman()`.